controlpanel/settings/common.py

```diff
@@ -46,6 +46,7 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "django_extensions",
```

Comment:
hmm,  have we installed this package?  🤔

Reply:
To be looked at at a later date